app/assets/sass/application.scss

```diff
@@ -757,4 +779,101 @@ ul.sub-tabs li a.active span.badge {
 .govuk-input__prefix {
     height: 40px;
     font-size: 19px;
-}
\ No newline at end of file
+}
+
+.view-claim-display-helper{
+    display: flex;
+    align-items: center;
+} 
+
+.justify-content-end {
+    justify-content: end;
+}
+
+.float-left {
+    float: left;
+}
+
+.text-italic {
+    font-style: italic;
+}
+
+.govuk-warning-text__text {
+    font-size: 19px;
+}
+
+.claim-table-email-text {
+    word-break: break-all;
+}
+
+.claim-table-width-one-third {
+    width: 33%;
+}
+
+.claim-table-width-one-sixth {
+    width: 16.66%;
+}
+
+.claim-table-width-full {
+    width: 100% !important;
+}
+
+.claim-table-error-text {
+    color: $govuk-error-colour!important;
+    padding-left: 15px;
+    border-left: 5px solid #d4351c;
+}
+
+.claim-table-deduction-section-container {
+    text-align: left;
+    align-items: center;
+    display: flex;
+}
+
+.claim-table-claim-section-container {
+    display: flex;
+    flex-direction: row-reverse;
+}
+
+.claim-table-claim-section-document-container {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+.claim-table-decision-error-margin {
+    margin-top: 30px!important;
```

Comment:
Ideally want to try and get around this as `!important` is not good practice.

Reply:
I tried a few alternative approaches but to no avail. Rule only gets set when a specific error pops up upon submitting a claim rejection - so hopefully okay as only a rare case?